mobileapps/tests.py

```diff
@@ -273,3 +269,84 @@ def test_mobileapps_inactive(self):
         self.assertEqual(response.status_code, 200)
         self.assertEqual(response.data['count'], 1)
         self.assertEqual(response.data['results'][0]['is_active'], False)
+
+    def test_mobileapps_users(self):
+        mobileapp = MobileApp.objects.create(
+            name='ABC App',
+            identifier='ABC',
+            operating_system=self.test_mobileapp_operating_system,
+            current_version=self.test_mobileapp_current_version,
+            updated_by=self.user,
+        )
+
+        users = UserFactory.create_batch(10)
+        mobileapp.users.add(*users)
+
+        response = self.do_get(reverse('mobileapps-users', kwargs={'mobileapp_id': mobileapp.id}))
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.data['count'], len(users))
+        self.assertEqual(len(response.data['results']), len(users))
+        self.assertEqual(response.data['num_pages'], 1)
+
+    def test_mobileapps_users_add(self):
+        mobileapp = MobileApp.objects.create(
+            name='ABC App',
+            identifier='ABC',
+            operating_system=self.test_mobileapp_operating_system,
+            current_version=self.test_mobileapp_current_version,
+            updated_by=self.user,
+        )
+
+        users = UserFactory.create_batch(5)
+        mobileapp.users.add(users[0])
+        mobileapp.users.add(users[1])
+
+        response = self.do_get(reverse('mobileapps-users', kwargs={'mobileapp_id': mobileapp.id}))
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.data['count'], 2)
+        self.assertEqual(len(response.data['results']), 2)
+        self.assertEqual(response.data['num_pages'], 1)
+
+        data = {
+            "user_ids": "{},{},{}".format(users[2].id, users[3].id, users[4].id)
+        }
+
+        response = self.do_post(reverse('mobileapps-users', kwargs={'mobileapp_id': mobileapp.id}), data=data)
+        self.assertEqual(response.status_code, 200)
+
+        response = self.do_get(reverse('mobileapps-users', kwargs={'mobileapp_id': mobileapp.id}))
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.data['count'], len(users))
+        self.assertEqual(len(response.data['results']), len(users))
+        self.assertEqual(response.data['num_pages'], 1)
+
+    def test_mobileapps_users_remove(self):
+        mobileapp = MobileApp.objects.create(
```

Comment:
add a `setupClass` method of the test case class and add these operations into that
```
 mobileapp = MobileApp.objects.create(
            name='ABC App',
            identifier='ABC',
            operating_system=self.test_mobileapp_operating_system,
            current_version=self.test_mobileapp_current_version,
            updated_by=self.user,
        )

        users = UserFactory.create_batch(5)
        mobileapp.users.add(*users)
```